blackbox/docs/config/node.rst

```diff
@@ -254,6 +254,19 @@ Plugins
 
   If any plugin listed here is missing, the CrateDB node will fail to start.
 
+CPU
+===
+
+**processors**
+  | *Runtime:* ``no``
+
+  The number of available processes is automatically guessed, and so most of
+  the time you will not need to configure this explicitly.
+
+  However, in some situations, such as when CrateDB is being run on top of
+  Docker, the number of processors may be guessed incorrectly. If this happens,
+  you can manually configure the number of processors using this setting.
+
```

Comment:
In addition I would mention as a second use-case that this setting makes sense to be set if this instance should only use a fraction of the CPU. This would be the case if you run multiple Crate instances on the same host system (like we do with Kubernetes in cratedb.cloud).